CodeStarterCamp_Week1/main.swift

```diff
@@ -3,8 +3,8 @@ import Foundation
 drawIceTop()
 drawIceBar()
 
-func drawPeperoBarLength(BarLength: Int) {
-    for _ in 1...BarLength {
+func drawPeperoBarLength(sickLength: Int) {
```

Comment:
헛 코멘트의 의도는 length 라는 단어가 불필요하게 반복되는 것 같아
```suggestion
func drawPeperoBar(length: Int) {
```
요런 식의 수정을 말씀드린 거였어요!

Reply:
제가 함수를 잘 못사용해서 그런지 저 부분을 length로 사용하면 body함수에 length로 적용이 되어 sixckLenght로 변경 하였습니다

Reply:
음 body 함수는 `drawPeperoBarLength` 함수와 스코프가 달라 영향을 안 줄텐데 이상하네요 🤔